### DISCUSSION TITLE
Collisions between timesteps

### DISCUSSION THREAD

**Discussion by Bradbev** [jrouwe/JoltPhysics#1335] [General]
My apologies if this is answered somewhere already and I missed it.  
Does Jolt support precision collision detection between timesteps?  For example, a bouncing ball is unlikely to hit the floor exactly on a timestep.  If I have a simulation that I want to save the state of and allow (interpolated) replay of that state, is there a way to get the collision information and time of collision?

Thanks!
Brad

**Comment by jrouwe** [comment]
If you set the motion quality to `LinearCast` then you should see the ball hit the floor exactly. Jolt doesn't simulate beyond the first collision in a time step, see [the documentation](https://jrouwe.github.io/JoltPhysics/#continuous-collision-detection). This means there's no special API for it.

**Reply by Bradbev** [reply]
Thanks very much, and sorry that I missed that in the documents :(
Awesome library by the way!
